client/ayon_core/hosts/houdini/plugins/create/create_redshift_rop.py

```diff
@@ -118,18 +104,45 @@ def remove_instances(self, instances):
 
         return super(CreateRedshiftROP, self).remove_instances(instances)
 
-    def get_pre_create_attr_defs(self):
-        attrs = super(CreateRedshiftROP, self).get_pre_create_attr_defs()
+    @staticmethod
+    def update_node_parameters(node, creator_attributes):
+        """update node parameters according to creator attributes.
+
+        Implementation of update_node_parameters.
+
+        Args:
+            node(hou.Node): Houdini node to apply changes to.
+            creator_attributes(dict): Dictionary of creator attributes.
+        """
+
+        ext = creator_attributes.get("image_format")
+        ext_format_index = {"exr": 0, "tif": 1, "jpg": 2, "png": 3}
+
+        parms = {"RS_outputFileFormat": ext_format_index[ext]}
+
+        multi_layered_mode = creator_attributes.get("multi_layered_mode")
+        multipart = False
+        if multi_layered_mode == "2":
+            multipart = True
+
+        if ext == "exr":
+            parms["RS_outputMultilayerMode"] = multi_layered_mode
+            parms["RS_aovMultipart"] = multipart
+
+        parms["RS_archive_enable"] = creator_attributes.get("split_render")
+
+        node.setParms(parms)
+
+    def get_instance_attr_defs(self):
         image_format_enum = [
             "exr", "tif", "jpg", "png",
         ]
-        multi_layered_mode = [
-            "No Multi-Layered EXR File",
-            "Full Multi-Layered EXR File"
-        ]
-
+        multi_layered_mode = {
+            "1": "No Multi-Layered EXR File",
+            "2": "Full Multi-Layered EXR File"
+        }
```

Comment:
Why is this suddenly a string of `"1"`  and `"2"`?

Reply:
This is the same as in this PR https://github.com/ynput/ayon-core/pull/328
to me it felt easier to use it in this way instead of adding a dictionary to map these strings to these values. 

from debugging side: I'm not sure if it's a good idea.

Reply:
So the actual Houdini parm takes a string number?